crates/re_renderer/src/renderer/rectangles.rs

```diff
@@ -653,3 +677,90 @@ impl Renderer for RectangleRenderer {
         ]
     }
 }
+
+fn is_float_filterable(format: &wgpu::TextureFormat) -> bool {
+    // based on https://gpuweb.github.io/gpuweb/#texture-format-caps,
+    // TODO(emilk): add this to the wgpu crate
```

Comment:
Latest wgpu has this already. TextureFormat::describe is gone/refactored there. We will be able to use 
https://github.com/gfx-rs/wgpu/blob/trunk/wgpu-types/src/lib.rs